lib/index.js

```diff
@@ -17,22 +17,25 @@ exports.start = (options) => {
 
     const output = (str) => options.out.write(`${str}\n`);
     const colors = Print.colors(options.colors);
-    const ctx = { options, colors };
+    const ctx = { options, colors, output, DisplayError };
 
     output(''); // Just a newline to make some room
 
-    if (args instanceof Error) {
-        throw new DisplayError(`${internals.usage(ctx)}\n\n` + colors.red(args.message));
-    }
+    const commandAndPart = (args instanceof Error) ? options.argv[2] : args._[2];
+    const extraArgs = (commandAndPart === 'run') ? options.argv : args._;
```

Comment:
Did you get the `_` as prop convention from anywhere? Just curious

Reply:
It's part of the bossy output: https://github.com/hapijs/bossy/blob/master/test/index.js#L80.  It consists of all the leftover argvs after parsing-out options in the bossy definition.